pkg/materialize/source_postgres.go

```diff
@@ -86,24 +80,17 @@ func (b *SourcePostgresBuilder) Create() error {
 
 	q.WriteString(fmt.Sprintf(` (%s)`, p))
 
-	if len(b.table) > 0 {
-		q.WriteString(` FOR TABLES (`)
-		for i, t := range b.table {
-			if t.Alias == "" {
-				t.Alias = t.Name
-			}
-			q.WriteString(fmt.Sprintf(`%s AS %s`, t.Name, t.Alias))
-			if i < len(b.table)-1 {
-				q.WriteString(`, `)
-			}
+	q.WriteString(` FOR TABLES (`)
+	for i, t := range b.table {
+		if t.Alias == "" {
+			t.Alias = t.Name
+		}
+		q.WriteString(fmt.Sprintf(`%s AS %s`, t.Name, t.Alias))
```

Comment:
@bobbyiliev it just struck me that I think we're mishandling special characters here. We should be splitting out separate fields for `schema_name` and `alias_schema`, and then using `QuoteString` here.

Reply:
This indeed seems to be the case. We are semi-handling this only in the [AddSubsource func](https://github.com/MaterializeInc/terraform-provider-materialize/pull/487/files/d24cfe8fa1a8368675c134dccd25a3f93c62efee#diff-c687ee9b62a59bc742cf585af291cacc0ebac5ac171a43a978618648c77fdd40L118-L125).

Is your idea to introduce two separate attributes as follows:

```hcl
  table {
    name  = "some_table"
    schema_name = "public"
    alias = "alias_table_name"
    alias_schema = "public"
  }
  ```

Reply:
Yes! Although as @sploiselle mentioned on Slack, this is our chance to adjust the naming of these attributes. We could alternatively call the attributes `upstream_name`, `upstream_schema_name`, `name`, and `schema_name`.